uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestActivity.kt

```diff
@@ -213,7 +184,7 @@ class BookingRequestView @JvmOverloads constructor(context: Context,
     override fun bindPriceAndEta(quote: Quote, card: String) {
         val currency = Currency.getInstance(quote.price.currencyCode)
 
-        bookingRequestPriceWidget?.bindViews(quote, context.getString(R.string.kh_uisdk_estimated_arrival_time), currency)
+        bookingRequestPriceWidget?.bindViews(quote, this.getString(R.string.kh_uisdk_estimated_arrival_time), currency)
```

Comment:
we can also remove the `this` from the getString

Reply:
Done